### DISCUSSION TITLE
How not to `forcePull`?

### DISCUSSION THREAD

**Discussion by dimaqq** [nektos/act#2338] [General]
`  -p, --pull                                              pull docker image(s) even if already present (default true)`

for some reason, I don't want that, how to unset this flag?

I tried `--no-pull` in `.actrc` but then act fails complaining that there's no such flag.

**Comment by dimaqq** [comment]
Thanks to ChatGPT, apparently it's `--pull=false`

**Comment by surgiie** [comment]
`--pull=false` doesnt work for me. `act` still tries to pull from docker hub/remote.
